assets/src/edit-story/components/panels/design/layer/constants.js

```diff
@@ -15,4 +15,4 @@
  */
 
 export const DEFAULT_LAYERS_VISIBLE = 6;
-export const LAYER_HEIGHT = 40;
+export const LAYER_HEIGHT = 44;
```

Comment:
I'm seeing 48 in figma for layer height?
<img width="268" alt="Screen Shot 2021-03-04 at 1 24 09 PM" src="https://user-images.githubusercontent.com/10720454/110025662-ec172d00-7cec-11eb-980e-d053cdaec830.png">

Reply:
Oh nice catch. Looks like I missed that

The layers are 44px so I'll need to change just the height of just the header instead of the layer height (which is used by all the buttons as well)
![image](https://user-images.githubusercontent.com/22185279/110028633-b83e0680-7cf0-11eb-9809-aa473e1c94d7.png)

Reply:
Thanks for updating!